README.md

```diff
@@ -44,20 +44,56 @@ If you want to install this package as dependency, you can install it from this
 npm install --save https://github.com/HatScripts/circle-flags
 ```
 
+### Custom builds
+
+You can create a customized build by running the included script:
+
+```
+$ npm install
+$ CORNER_RADIUS=25% npm run build
+```
+
+This will generate flags with rounded corners instead of the default circle. The files will appear in the `build/` directory.
```

Comment:
Just a small tweak :)

```suggestion
This will generate flags as squares with rounded corners instead of the default circle. The files will appear in the `build/` directory.
```